website/docs/tutorials/07-testing/02-cheatcodes/example.md

```diff
@@ -0,0 +1,44 @@
+# `example`
+
+```python
+def example(self, *args: Any, **kwargs: Any) -> None:
+```
+
+This is an addition to the [`given`](./given.md) cheatcode, but, due to how the protostar works,
+`given` cheatcode does not have to be used in order to invoke `example`.
+While `given` runs tests with multiple, randomly generated data sets,
+`example` takes certain values and runs tests using them.
+Tests with the data specified in `example` will always be run before applying the data from `given`.
+
+Multiple `example`s can be used for one test. In such a case, all of them will be run sequentially,
+only after that allowing `given` to run.
+
+`example` is not limited by [`max_examples`](./max-examples.md) and is not connected to it in any way.
+
+:::warning
+This cheatcode is only available in [setup cases](../README.md#setup-case).
+:::
+
+```cairo title="Example"
+@external
+func setup_less_equal_compare() {
+    %{
+        example(1, 2)
+        example(2, 3)
+
+        given(a = strategy.integers(15, 20), b = strategy.integers(10, 14))
+
+        example(a=3, b=4)
+        example(b=6, a=5)
+    %}
+    return ();
+}
+
+@external
+func test_examples{syscall_ptr: felt*, range_check_ptr}(a: felt, b: felt) {
+    assert_le(0, a);
```

Comment:
[Nitpick] Maybe it is worth to include it:
```suggestion
    # Tested against:
    # (1,2)
    # (3,4)
    # fuzzing: a = strategy.integers(15, 20), b = strategy.integers(10, 14) 
    assert_le(0, a);
```

Reply:
I didn't see it in other docs (but maybe I didn't look close enough), but I don't mind adding it.

Reply:
[update docs](https://github.com/software-mansion/protostar/pull/947/commits/75b02de8f52d7eced8f64ae8687db4947c59bb45)